decidim/decidim
# Fix deleted user error in schema

#### :tophat: What? Why?
While reviewing #13666, i have accidentally entered in a meeting where the author has deleted its account, and as a result i got the classic error when the author is missing. 

Additionally, I have tried to remove the duplication formed around the profile_url method. 

<details>
<summary>Stacktrace</summary>

```
17:59:01 web.1         | ActionView::Template::Error (No route matches {:action=>"show", :controller=>"decidim/profiles", :nickname=>""}, possible unmatched constraints: [:nickname]):
17:59:01 web.1         |     1: <script type="application/ld+json">
17:59:01 web.1         |     2: <%== render_schema_org_event_meeting(meeting) %>
17:59:01 web.1         |     3: </script>
17:59:01 web.1         |   
17:59:01 web.1         | actionpack (7.0.8.4) lib/action_dispatch/journey/formatter.rb:44:in `path'
17:59:01 web.1         | actionpack (7.0.8.4) lib/action_dispatch/routing/route_set.rb:824:in `url_for'
17:59:01 web.1         | actionpack (7.0.8.4) lib/action_dispatch/routing/route_set.rb:271:in `call'
17:59:01 web.1         | actionpack (7.0.8.4) lib/action_dispatch/routing/route_set.rb:327:in `block in define_url_helper'
17:59:01 web.1         | actionpack (7.0.8.4) lib/action_dispatch/routing/routes_proxy.rb:45:in `profile_url'
17:59:01 web.1         | /home/alecslupu/Sites/decidim/redesign/decidim-core/lib/decidim/engine_router.rb:50:in `method_missing'
17:59:01 web.1         | /home/alecslupu/Sites/decidim/redesign/decidim-meetings/lib/decidim/meetings/schema_org_event_meeting_serializer.rb:72:in `organizer_user'
17:59:01 web.1         | /home/alecslupu/Sites/decidim/redesign/decidim-meetings/lib/decidim/meetings/schema_org_event_meeting_serializer.rb:48:in `organizer'
17:59:01 web.1         | /home/alecslupu/Sites/decidim/redesign/decidim-meetings/lib/decidim/meetings/schema_org_event_meeting_serializer.rb:27:in `serialize'
17:59:01 web.1         | /home/alecslupu/Sites/decidim/redesign/decidim-core/app/serializers/decidim/exporters/serializer.rb:24:in `run'
17:59:01 web.1         | /home/alecslupu/Sites/decidim/redesign/decidim-core/lib/decidim/exporters/json.rb:16:in `block in export'
17:59:01 web.1         | /home/alecslupu/Sites/decidim/redesign/decidim-core/lib/decidim/exporters/json.rb:15:in `map'
17:59:01 web.1         | /home/alecslupu/Sites/decidim/redesign/decidim-core/lib/decidim/exporters/json.rb:15:in `export'
17:59:01 web.1         | /home/alecslupu/Sites/decidim/redesign/decidim-meetings/app/helpers/decidim/meetings/meetings_helper.rb:147:in `render_schema_org_event_meeting'
17:59:01 web.1         | /home/alecslupu/Sites/decidim/redesign/decidim-meetings/app/views/decidim/meetings/meetings/_schema_org_event_meeting.html.erb:2
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/base.rb:244:in `public_send'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/base.rb:244:in `_run'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/template.rb:157:in `block in render'
17:59:02 web.1         | activesupport (7.0.8.4) lib/active_support/notifications.rb:208:in `instrument'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/template.rb:361:in `instrument_render_template'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/template.rb:155:in `render'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/renderer/partial_renderer.rb:251:in `block in render_partial_template'
17:59:02 web.1         | activesupport (7.0.8.4) lib/active_support/notifications.rb:206:in `block in instrument'
17:59:02 web.1         | activesupport (7.0.8.4) lib/active_support/notifications/instrumenter.rb:24:in `instrument'
17:59:02 web.1         | activesupport (7.0.8.4) lib/active_support/notifications.rb:206:in `instrument'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/renderer/partial_renderer.rb:246:in `render_partial_template'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/renderer/partial_renderer.rb:237:in `render'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/renderer/renderer.rb:81:in `render_partial_to_object'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/renderer/renderer.rb:27:in `render_to_object'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/renderer/renderer.rb:22:in `render'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/helpers/rendering_helper.rb:37:in `block in render'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/base.rb:270:in `in_rendering_context'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/helpers/rendering_helper.rb:33:in `render'
17:59:02 web.1         | /home/alecslupu/Sites/decidim/redesign/decidim-meetings/app/views/decidim/meetings/meetings/_meeting.html.erb:3
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/helpers/capture_helper.rb:45:in `block in capture'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/helpers/capture_helper.rb:209:in `with_output_buffer'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/helpers/capture_helper.rb:45:in `capture'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/helpers/rendering_helper.rb:101:in `_layout_for'
17:59:02 web.1         | actionview (7.0.8.4) lib/action_view/renderer/partial_renderer.rb:252:in `block (2 levels) in render_partial_template'
17:59:02 web.1         | /home/alecslupu/Sites/decidim/redesign/decidim-core/app/views/layouts/decidim/shared/_layout_item.html.erb:10

```
 </details>

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #13406
- Related to #13439

#### Testing
1. Using a newly created user, create a meeting 
2. Visit the meeting page 
3. Delete the user account 
4. Visit the meeting page 
5. See the error 
6. Apply patch 
7. See error is gone.

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![image](https://github.com/user-attachments/assets/e18c96a8-a7a2-4058-bf50-bbe8772e09e0)

:hearts: Thank you!
